src/file_utils.mjs

```diff
@@ -9,33 +9,31 @@ import JSON5 from 'json5';
 const RESULTS_DIR = 'pixel_processing_results';
 
 /**
- * Get common parameters
+ * Read common parameters
  * @param {string} mainPixelDir - path to the main pixels directory
- * @param {boolean} forceLowerCase - whether to force to lowercase
  * @returns {object} common parameters
  */
-export function getCommonParams(mainPixelDir, forceLowerCase) {
```

Comment:
After thinking through case-sensitivity more, I think it makes more sense to keep it compartmentalized to LiveValidator.